README.md

```diff
@@ -1,24 +1,48 @@
-# README
+# Junk4Dollars
```

Comment:
This is looking great, however there are a couple of things that as a developer would want to do, but do not see documented.

It would probably improve the readme to add instructions on:
1. How to run the linter (Rubocup).
2. How to run the tests locally.